Code/Tools/ProjectManager/Source/Cli/O3deCliBindings.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+// Qt defines slots, which interferes with the use here.
+#pragma push_macro("slots")
+#undef slots
+#include <Python.h>
+#include <pybind11/pybind11.h>
+#pragma pop_macro("slots")
+
+namespace O3DE::ProjectManager
+{
+    //! Interface used to interact with the o3de cli python functions
+    class O3deCliBindings
```

Comment:
Though the code we are calling lives inside the o3de python cli application, I'm not a fan that the name O3deCliBindings seems to imply we are somehow using a command line interface or interacting with a command line.  There is no command line involved here, it's just the type of application o3de.py is.